core/kazoo_attachments/README.md

```diff
@@ -0,0 +1,56 @@
+# Kazoo Attachments
+
+This application is meant to be used through `crossbar`'s `/:api_version/accounts/:account_id/storage`
+endpoint. In case you want to use it directly, the recommended way is to use `kz_datamgr.erl` module from `core/kz_data` app.
+
+More information about attachments and storage [here](https://github.com/2600hz/kazoo/blob/master/applications/crossbar/doc/storage.md).
+
+## How to use it?
+
+Before you *put* or *fetch* an attachment, make sure you know the handler name and the required
+configuration settings for the storage service you want to use e.g: [dropbox](https://github.com/2600hz/kazoo/blob/master/applications/crossbar/doc/storage.md#storageattachmentdropbox).
+
+Put attachment to dropbox example:
+
+`kz_datamgr:put_attachment/5` parameters:
+- **DbName**: Database name where the attachment's parent document is stored.
+- **DocId**: ID of the document to which the attachment will belong to.
+- **AName**: Attachment's file name.
+- **Contents**: Attachment's content.
+- **Options**: Here is where you need to send the attachment handler name along the required configuration
+fields for the given handler.
+
+```erlang
+DbName = <<"db_name">>,
+DocId = <<"666cbad1ac4b41879c10dba638f8777b">>,
+AName = <<"dropbox_example_file.txt">>,
+Contents = <<"this is some dummy content for the dropbox file">>,
+% The `AttHandler' must match the storage service handler name (without the `.erl' part), in this case `kz_att_dropbox'.
+AttHandler = 'kz_att_dropbox',
+AttSettings = #{oauth_doc_id => <<"oauth_doc_id generated by the storage service provider">>}
+Options = [{'plan_override', #{'att_handler' => {AttHandler, AttSettings}
```

Comment:
`plan_override` should be given as an example

Reply:
this should focus on how to implement an attachment handler.
what values are expected (with samples) on the methods and how to return values.
the use of kz_datamgr here is out of scope

Reply:
you can provide an example of `kz_att_dummy_file` that just stores and reads to/from disk.